src/mgs/mgs_parser.rs

```diff
@@ -34,13 +35,40 @@ pub fn params() -> Vec<String> {
     .collect::<Vec<_>>()
 }
 
+pub fn name_only_params() -> Vec<String> {
+    vec!["mgs.MGS.live.*".to_string()]
```

Comment:
What does this look like if there are multiple MGS?

Reply:
I'm using a wildcard character instead of `MGS` now.